goon-universe/queries.js

```diff
@@ -540,12 +540,18 @@ function getStakeholderDescriptions(scenarioID){
 }
 
 function getInitReflectPage(scenarioID, callback){
-    let thisQuery= 'select prompt.prompt from pages, prompt where pages.id = prompt.page_id and pages.order = '+ INITIAL_REFLECTION +' and scenario_id = $1'
+    let thisQuery= 'select pages.body_text, prompt.prompt, prompt.prompt_num from pages, prompt where pages.id = prompt.page_id and pages.order = '+ INITIAL_REFLECTION +' and scenario_id = $1'
     pool.query(thisQuery, [scenarioID], (error,results) => {
         if (error) {
             throw error
         }
-        callback(results.rows)
+        let response = {}
+        response.prompts = results.rows.map(row => ({
```

Comment:
Pinging @miesoz to make sure he is aware of the format change here

Reply:
It looks good to me. I've tested it this way and it seems to work fine

Reply:
Looked over it with Kunal, seems good.